libs/ui/src/lib/status-badge/status-badge.component.html

```diff
@@ -1 +1,20 @@
-{{ label }}
+<div>
+  {{ label }}
+  <cvi-ng-icon
+    *ngIf="withTooltip"
+    svgClass="badge-icon"
+    [cviNgTooltip]="tooltipText ? tooltipText : ''"
+    dataAttribute="test-tooltip-source"
+  >
+    <svg
```

Comment:
A correct way to use `cvi-ng-icon` is to simply provide a `name` prop to it. No need to insert SVG code directly. If you need a new icon, you can add it to the `icons` package. However the "info" icon is already there: https://e-gov.github.io/cvi/?path=/story/angular-icon--default&args=name:info. 

This would also make the status badge component more flexible, giving a user an opportunity to use different icons (at least in the design i see not only the "info" icon can be used but also a "check" icon).